.rubocop_todo.yml

```diff
@@ -0,0 +1,33 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2019-10-08 20:34:01 +0300 using RuboCop version 0.75.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Performance/RegexpMatch:
+  Exclude:
+    - 'lib/yavdb/sources/snyk_io.rb'
+
+# Offense count: 1
+# Configuration parameters: Max.
+RSpec/ExampleLength:
```

Comment:
In general the tests are not amazing but for now it was the best way to do it.
I have some ideas to do more unit tests and reduce the integration and complicated ones.
I am okay in leaving this as is for now, but I am also open for suggestions.